app-backend/tile/src/main/scala/routes/SceneRoutes.scala

```diff
@@ -15,19 +15,19 @@ import akka.http.scaladsl.server._
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.model.{ContentType, HttpEntity, HttpResponse, MediaTypes}
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.dispatch.MessageDispatcher
 import spray.json._
 import com.typesafe.scalalogging.LazyLogging
 import cats.data._
 import cats.implicits._
-import java.util.UUID
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import java.util.UUID
 import scala.concurrent._
 import scala.util._
 
 object SceneRoutes extends LazyLogging with KamonTraceDirectives {
 
-  def root: Route =
+  def root(implicit dispatcher: MessageDispatcher): Route =
```

Comment:
Should we just define the thumbnail I/O dispatcher here vs. pulling it in from the `Router`?